waltz-server/src/main/java/com/wepay/waltz/server/internal/Partition.java

```diff
@@ -428,7 +439,10 @@ private void initializeFeed(MountRequest request, PartitionClient client) {
 
             if (fetchSize == 0) {
                 // The client is up to date. Send the response immediately. No need to enqueue a feed context.
-                client.sendMessage(response, true);
+                if (!client.sendMessage(response, true)) {
+                    logger.info(String.format("Unable to send mount response message. ClientId: %s, SeqNum: %d, RequestId: %s",
```

Comment:
I think it would be better if we mention clearly that the message is not able to send as the Channel ctx is null.
Just a suggestion. That way it will be easier for someone looking at the log statement to understand clearly.